recipe/meta.yaml

```diff
@@ -1,30 +1,32 @@
 {% set name = "dask-yarn" %}
-{% set version = "0.8.1" %}
+{% set version = "0.9" %}
+{% set pkg_loc = "py2.py3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 44ba7cd2e2ca6004f8cbcc358816fea68a898da509cd41dd85e7c4a7c7fcb4ed
+  url: https://files.pythonhosted.org/packages/{{ pkg_loc }}/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-{{ pkg_loc }}-none-any.whl
```

Comment:
There should be an `sdist` on PyPI. They also included a `sha256` checksum for it. Would recommend using the `sdist` instead (unless there is a reason not to)

If the templated URL is out-of-date here, would look at [this example in `staged-recipes`]( https://github.com/conda-forge/staged-recipes/blob/7033c8924612d09c90e60ed6eda6f2986414d94b/recipes/example/meta.yaml#L14 )